data/config-schema/src/main/resources/net/corda/schema/configuration/flow/1.0/corda.flow.json

```diff
@@ -33,7 +33,7 @@
           "default": 16000
         },
         "maxFlowSleepDuration": {
-          "description": "The maximum delay in milliseconds before Corda schedules a periodic wake-up.",
+          "description": "Deprecated. The maximum delay in milliseconds before Corda schedules a periodic wake-up.",
```

Comment:
Is this the official way we are to deprecate configs? Just curious as i know some other ones which need deprecating.